collection-scripts/gather_images

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+IFS=$'\n'
+
+BASE_COLLECTION_PATH="/must-gather"
+
+IMAGES_PATH=${BASE_COLLECTION_PATH}/cluster-scoped-resources/images
+NAMESPACE_PATH=${BASE_COLLECTION_PATH}/namespaces
+
+for name in $(oc get image -o=custom-columns=NAME:.metadata.name --no-headers)
+do
+    oc get image $name -o yaml >> ${IMAGES_PATH}
+    echo '---------------' >> ${IMAGES_PATH}
+done
+
+for line in $(oc get imagestreams -o=custom-columns=NAMESPACE:.metadata.namespace,NAME:.metadata.name --no-headers --all-namespaces)
+do
+    namespace=$(echo $line | awk -F ' ' '{print $1}')
+    name=$(echo $line | awk -F ' ' '{print $2}')
```

Comment:
we can reduce number of pipes and awks by simply:
```
OLD_IFS=$IFS
IFS=' ' read namespace name <<< $line
IFS=$OLD_IFS
```

Reply:
this is something we should apply to other places as well since we use the same pattern in for loops

Reply:
Done